packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -155,6 +155,9 @@
     "localVideoMovementLabel": "Movable Local Video Tile",
     "localVideoSelectedDescription": "{cameraName} selected"
   },
+  "dialpad": {
+    "defaultText": "Enter a number"
```

Comment:
nit: here and the prop this is put into should `placeholderText` instead of defaultText.
placeholder is what is there when there is no input, and isn't live text (if you hit submit on a form, placeholder text is not submitted). Whereas default text is live text that the input box should start with